components/brave_rewards/resources/tip/components/tipSite.tsx

```diff
@@ -23,7 +24,17 @@ class TipSite extends React.Component<Props, {}> {
   }
 
   onTweet = () => {
-    this.actions.onTweet(this.props.publisher.name, '')
+    const url = this.props.url
```

Comment:
Last thing 😃 I think that this check needs to go into provider check for twitter as some other providers maybe will not need url and will just have name

Reply:
Makes sense, fixed.